### ISSUE TITLE
Add contact:facebook (instagram, twitter..)

### ISSUE THREAD

**Issue by thibaultmol** [openstreetmap/id-tagging-schema#434]
Hi, when adding businesses to OSM, most of them have Facebook, Instagram, Twitter or others.
Because of that, it probably makes sense to have that be an option in the interface for people to add.
(I already do this manually, but for people that prefer using the UI to select tags, it should be considered to show up there as well)

https://wiki.openstreetmap.org/wiki/Key:contact

**Comment by danieldegroot2** [comment]
_This is a duplicate of #121 ._ (ignore the facebook `website` part)

**Comment by tyrasd** [comment]
it's related, but not really a duplicate, since issue #121 is dealing with the different ways to map social media data (e.g. `contact:facebook=*` vs. `facebook=*`). At the moment the tagging presets don't include any social media fields.

**Comment by riQQ** [comment]
`contact:facebook` was added in #859 and is so far the only social media field.

**Comment by tiptoptom** [comment]
Would you accept a PR for [contact:mastodon=*](https://wiki.openstreetmap.org/wiki/Key:contact:mastodon)?

https://taghistory.raifer.tech/?#***/contact%3Amastodon/

**Comment by matkoniecz** [comment]
Yes, group issues are terrible. You cannot close them with a single PR, you never know whether it is now complete and you need to read through already handled discussions.

**Comment by tiptoptom** [comment]
I've opened #2234 for mastodon.
